### PR TITLE
gloss-awe:0.0.3

### PR DESCRIPTION
**Pull request by RolfBremer**
I am submitting
- [x] a new package
- [ ] an update for a package

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name in conformance with the guidelines
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] ensured that my submission does not infringe upon the rights of a third party
- [x] tested my package locally on my system and it worked
- [x] named this PR as `name:version` of the submitted package
- [x] agree that my package will not be removed without good reason

Description: This typst component creates a glossary page from a given pool of potential glossary
entries using only those entries, that are marked with the `gls` or `gls-add` functions in
the document. See the `README.md` for examples and usage.

### CODE REVIEW

**Comment by reknih** [comment]
Thank you for your submission! You will need to choose a different name, because just `glossary` is plainly descriptive. Your name can be descriptive to the function of the package, however, it also needs to have some of your "special creative sauce" because users may perceive the package as officially endorsed otherwise.

**Comment by RolfBremer** [comment]
Ok, added some sauce to the name. :-)
Ok so?

File: packages/preview/gloss-awe-0.0.3/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# typst-glossary
```
Comment (laurmaedje) [inline packages/preview/gloss-awe-0.0.3/README.md:1]: this should be `gloss-awe`

File: packages/preview/gloss-awe-0.0.3/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+entries using only those entries, that are marked with the `gls` or `gls-add` functions in
+the document. See sample-usage document for details.
+
+⚠️ Typst is in beta and evolving, and this package evolves with it. As a result, no
+backward compatibility is guaranteed yet. Also, the package itself is under development
+and fine-tuning.
+
+## Marking the Entries
+
+To include a term into the glossary, it can be marked with the `gls` function. The
```
Comment (laurmaedje) [inline packages/preview/gloss-awe-0.0.3/README.md:15]: I think this README would benefit a lot from one integrated example where the glossary pool is maybe also in the main file. I played with it a bit and it took some a bit of time to figure out how the pieces fit together.

File: packages/preview/gloss-awe-0.0.3/README.md (hunk 1 above)
Comment (RolfBremer) [reply packages/preview/gloss-awe-0.0.3/README.md]: Done.

File: packages/preview/gloss-awe-0.0.3/README.md (hunk 2 above)
Comment (RolfBremer) [reply packages/preview/gloss-awe-0.0.3/README.md:15]: I agree, and added a complete sample and restructured the readme accordingly.

**Comment by laurmaedje** [comment]
Thank you!

**Pull request merged**